components/calendar/src/cal/hijri.rs

```diff
@@ -57,6 +57,19 @@ mod ummalqura_data;
 ///
 /// There are either 6 or 7 30-day months, so the length of the year is 354 or 355 days.
 ///
+/// # Crescent moon visibility
+///
+/// Islam says that months begin when an observer first sees the crescent moon.
+///
+/// For centuries, astronomers have been developing criteria for predicting crescent moon
+/// visibility. However, in most regions, atmospheric phenomena can impact visibility, making
+/// such predictions only an approximation of ground truth. Hijri calendar applications should
+/// therefore include the ability to specify a crescent sighting adjustment.
+///
+/// The primary exception is Saudi Arabia, where the KACST uses sophisticated telescopes to
```

Comment:
what do sophisticated telescopes and observations have to do with UAQ? it's an astronomical calculation

Reply:
They still claim to do observations. The calculations predict what their fancy infrared telescopes will see. I don't know what happens if they can't observe their prediction; I think they just claim that it won't ever happen, and they go to extreme measures to make sure that the crescent is visible when they claim it will be.

This is a good article on the subject:

https://www.middleeasteye.net/news/eid-2025-does-saudi-arabia-report-impossible-moonsightings

Here's an example image of a crescent sighting done with this method:

https://x.com/MulhamH/status/1927472335562375426

Reply:
ok, fair point. I changed it to: "The primary exception is Saudi Arabia, where the KACST publishes official predictions of crescent timings for multiple centuries."

Moved to #7330